test/testground/compositions/standard/plan.toml

```diff
@@ -1,39 +1,40 @@
 [metadata]
-name = "standard"
```

Comment:
Why did you remove this?

Reply:
it wasn't being used anywhere afaict. testground always refers to tests using a unique identifier